src/parameterizations/vertical/MOM_vert_friction.F90

```diff
@@ -1917,6 +2108,66 @@ subroutine vertvisc_init(MIS, Time, G, GV, US, param_file, diag, ADp, dirs, &
                  "The background kinematic viscosity in the interior. "//&
                  "The molecular value, ~1e-6 m2 s-1, may be used.", &
                  units="m2 s-1", fail_if_missing=.true., scale=US%m2_s_to_Z2_T)
+  call get_param(param_file, mdl, "USE_GL90_IN_SSW", CS%use_GL90_in_SSW, &
+                 "If true, use simpler method to calculate 1/N^2 in GL90 vertical "// &
+                 "viscosity coefficient. This method is valid in stacked shallow water mode.", &
+                 default=.false.)
+  call get_param(param_file, mdl, "KD_GL90", CS%kappa_gl90, &
```

Comment:
Please add arguments like `do_not_log=.not.CSuse_GL90_in_SSW`  to this and any of the following get_param calls that are for any variables that are only used when GL90 is being used, so as to avoid creating overly large MOM_parameter_doc files for cases that are not using GL90 and for which any of these parameters are therefore unused.

Reply:
Good point. I opted to insert an if-statement to not even read all these parameters if `.not. CS%use_GL90_in_SSW`. Saves maybe a few milliseconds? Please let me know if the `do_not_log` approach if preferable for some reason.

Reply:
The overall if statement is fine, and as you note it will be slightly faster.  The only reason why we might prefer the do_not_log approach is that we have the ability to detect unread parameters and optionally bring down the model if there are any unread parameters in the MOM_input or MOM_override files.  Reading these parameters with the do_not_log flag allows us to use an override to the setting of CS%use_GL90_in_SSW to disable the scheme (e.g., in a MOM_override file) without having to comment out any of the subsidiary parameters.  The use of this do_not_log construct is far from universal in the MOM6 code, though, so we are not close to being able to make the default for `FATAL_UNUSED_PARAMS = True`, as we probably should.

I am happy to go with the code as revised, but just be aware that it is possible that at some point we will probably revise this new code as a part of sweeping changes to allow us to change the default for `FATAL_UNUSED_PARAMS`.

Reply:
Thanks for the explanation! I switched to the `do_not_log` approach.

Reply:
I should mention that for three GL90-related variables I chose `do_not_log = .true.`, rather than `do_not_log = .not. CS%use_GL90_in_SSW`:
* INPUTDIR
* KD_GL90_FILE
* KD_GL90_VARIABLE

I have seen it elsewhere in the code that `INPUTDIR` is not being logged. Not logging the other 2 variables either seemed consistent with that choice. I'm happy to use `do_not_log = .not. CS%use_GL90_in_SSW` for these 3 variables, too, though.

Reply:
INPUTDIR is not being logged in some places because we know that it will be logged elsewhere, and we were trying to manage where it shows up in the MOM_parameter_doc files and does not jump around too much as various module are turned on or off. By contrast, I do not think that the parameters KD_GL90_FILE or KD_GL90_VARIABLE occur anywhere else in the code, so it is important that they are logged here if they are active.

Reply:
Gotcha, thanks for clarifying. Changed.